conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -172,7 +177,18 @@ types:
           httpPath: HttpPath
           auth: optional<AuthType>
           args: list<ArgumentDefinition>
+          # How do we make it clear that the "errors" should be specifically related to the handling of the endpoint
+          #  (e.g. we don't want people adding AuthError to nearly every endpoint)?
+          # responses:
+          #   union:
+          #     - success: optional<Type>
+          #     - errors: set<TypeName>
           returns: optional<Type>
+          # TODO(pm): put up commit where ErrorName is alias for TypeName.
+          # TODO(pm): Maybe this should actually be `TypeName | ErrorDefinition` to allow devs to create new errors
+          #  inline. Think about if we want to disallow this behavior.
+          #  Let's not allow error types to be imported from other languages. Only Conjure-imports allowed.
+          errors: set<TypeName>
```

Comment:
Do we want to associate each of the `errors` with some documentation? We do this with `returns` already.